doc/deployment/README.md

```diff
@@ -101,7 +128,7 @@ and run:
 docker run \
     --network host \
     emcmongoose/<IMAGE>:debug \
-    [<ARGS>]
+    [<MONGOOSE CLI ARGS>]
 ```
 
 # Kubernetes
```

Comment:
line 140 - Helm not Melm

Reply:
line 143 - if you move manifests to other dir, don't forget to change here

Reply:
line 137 - consider stating explicitly: the only officially supported way to deploy Mongoose in Kubernetes is through Helm charts.